### PR TITLE
[sdf6] Changelog links to BitBucket backup

### PR DESCRIPTION
**Pull request by chapulina**
Signed-off-by: Louise Poubel <louise@openrobotics.org>

### CODE REVIEW

**Comment by j-rivero** [comment]
run tests

**Comment by j-rivero** [comment]
@osrf-jenkins run tests

**Comment by j-rivero** [comment]
@osrf-jenkins run tests

File: Changelog.md [hunk 1]
```diff
@@ -175,337 +175,337 @@
...
+    * [BitBucket pull request 48](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/48/fix-for-issue-22-reducing-inertia-across/diff)
 1. Add <use_terrain_paging> element to support terrain paging in gazebo
-    * [Pull request 47](https://bitbucket.org/osrf/sdformat/pull-request/47/add-element-inside-heightmap/diff)
+    * [BitBucket pull request 47](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/47/add-element-inside-heightmap/diff)
 1. Further reduce console output when using URDF models
-    * [Pull request 46](https://bitbucket.org/osrf/sdformat/pull-request/46/convert-a-few-more-sdfwarns-to-sdflog-fix/diff)
-    * [Commit](https://bitbucket.org/osrf/sdformat/commits/b15d5a1ecc57abee6691618d02d59bbc3d1b84dc)
+    * [BitBucket pull request 46](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/46/convert-a-few-more-sdfwarns-to-sdflog-fix/diff)
+    * [Commit](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/commits/b15d5a1ecc57abee6691618d02d59bbc3d1b84dc)
```
Comment (azeey) [inline Changelog.md:486]: This didn't work for me. Not sure how we map commit hashes.

File: Changelog.md [hunk 2]
```diff
@@ -175,337 +175,337 @@
...
+    * [BitBucket pull request 322](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/322)
 
 1. Enable coverage testing
-    * [Pull request 317](https://bitbucket.org/osrf/sdformat/pull-request/317)
+    * [BitBucket pull request 317](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/317)
 
 1. Add `friction_model` parameter to ode solver
-    * [Pull request 294](https://bitbucket.org/osrf/sdformat/pull-request/294)
+    * [BitBucket pull request 294](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/294)
     * [Gazebo pull request 1522](https://bitbucket.org/osrf/gazebo/pull-request/1522)
```
Comment (azeey) [inline Changelog.md:223]: Did you leave this out on purpose?

File: Changelog.md (hunk 1 above)
Comment (chapulina) [reply Changelog.md:486]: That's a valid link for me. From that page, you need to click on `View on GitHub` on the right.

File: Changelog.md (hunk 2 above)
Comment (chapulina) [reply Changelog.md]: Yeah it was on purpose, but then I realized it wasn't worth it coming back just to fix those, so the latest pass addresses that as well: 16dcc94 .

The link itself will only become valid once `osrf/gazebo` has been fully migrated.

**Comment by chapulina** [comment]
@azeey , the latest commit fixes additional links, as well as indentation on the changelog: 16dcc94

File: Changelog.md (hunk 1 above)
Comment (azeey) [reply Changelog.md:486]: ah, when I clicked it from my editor, it added an extra parenthesis. Works fine.

**Comment by osrf-triage** [comment]
This pull request has been mentioned on **Gazebo Community**. There might be relevant details there:

https://community.gazebosim.org/t/new-ignition-releases-2022-03-25-fortress-edifice-citadel/1343/1

**Pull request merged**